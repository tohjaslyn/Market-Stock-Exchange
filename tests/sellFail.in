sell trader1 ABC 10 2
add trader1 100
import trader1 ABC 10
sell trader1 ABC 12 1
exit
