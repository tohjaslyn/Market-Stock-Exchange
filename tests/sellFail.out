$ No such trader in the market.
$ Success.
$ Trader now has 10.00 units of ABC.
$ Order could not be placed onto the market.
$ Have a nice day.
